xamarin/Xamarin.Forms
# [Android, iOS] Fix issue rendering Shapes inside SwipeItemView

### Description of Change ###

Fix issue rendering Shapes inside `SwipeItemView.`

### Issues Resolved ### 

- fixes #12911 
- fixes #13096

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Android
![fix12911-droid](https://user-images.githubusercontent.com/6755973/99787984-bfb21900-2b20-11eb-9dea-c17d0f196c5e.gif)

iOS
![fix12911-ios](https://user-images.githubusercontent.com/6755973/99787445-0b17f780-2b20-11eb-8ef8-9054f130303e.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12911. Open any SwipeView (left or right), if can see the Red Shape, the test has passed.

### PR Checklist ###
